core/lib/dal/migrations/20221215094205_prover-job-identity.down.sql

```diff
@@ -1,2 +1,2 @@
 ALTER TABLE prover_jobs
-    DROP COLUMN proccesed_by;
\ No newline at end of file
+    DROP COLUMN processed_by;
```

Comment:
Please exclude the existing migration files that have typos. We can't touch those anymore.

Reply:
Yeah,sir,I will do this. Btw,can I add codespell check to workflow to integrate it

Reply:
Yes, I left a comment above

Reply:
I have added a relatively simple filtering file to exclude certain words. 

If you find it meaningful, you can modify the filtering conditions according to your needs.

skip:Files to exclude from the check
words.txt: Words to exclude from the check

Reply:
Yes, these should be reverted and added to skip I am guessing